.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,14 +17,23 @@ https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com
 ## Description
 Closes <ticket>
 
-## Testing done
```

Comment:
This is being mentioned in the [Configuring this pull request](https://github.com/department-of-veterans-affairs/component-library/blob/6b62b3e7d3f4eebcee5ceb3fe64bb7d46531c279/.github/PULL_REQUEST_TEMPLATE.md#configuring-this-pull-request) section (the second checkbox). So we probably need to remove that line now with such a robust QA checklist below now.